test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -109,6 +109,7 @@
     ("aten::nanquantile", datetime.date(2022, 9, 30)),
     ("aten::_convolution_double_backward", datetime.date(2022, 3, 31)),
     ("aten::_scatter_reduce", datetime.date(2022, 1, 31)),
+    ("aten::native_multi_head_self_attention", datetime.date(9999, 1, 1)),
```

Comment:
I don't think we should add these indefinite windows to the allow list

Reply:
Sorry this is a bug. It shouldn't be here